docker/caffe2/jenkins/build.sh

```diff
@@ -76,7 +76,7 @@ if [ -n "${JENKINS:-}" ]; then
 fi
 
 # Build image
-docker build \
+docker build $(if [[ "$image" == *rocm* ]]; then echo "--memory-swap -1"; fi) \
```

Comment:
Doesn't this control the swap memory available when building the docker image, not when running it?